applications/services/gui/view_dispatcher.h

```diff
@@ -123,13 +123,30 @@ void view_dispatcher_stop(ViewDispatcher* view_dispatcher);
  */
 void view_dispatcher_add_view(ViewDispatcher* view_dispatcher, uint32_t view_id, View* view);
 
+/** Try to add view to ViewDispatcher
+ *
+ * @param      view_dispatcher  ViewDispatcher instance
+ * @param      view_id          View id to register
+ * @param      view             View instance
+ * @return     {@code true} if there is no view by this ID and {@code false} otherwise
+ */
+bool view_dispatcher_try_add_view(ViewDispatcher* view_dispatcher, uint32_t view_id, View* view);
```

Comment:
Hi @JarvisCraft to be honest we don't use `try` pattern. But we'll accept something like `_view_exists()`.

Reply:
I like the idea of adding `_view_exists`, but I think that having `_try` methods for these specific methods is also essential since it allows the caller to not perform value lookup twice.

I am not suggesting adding `_try`-equivalents for each and every use-case, but IMO it is appropriate here since the `check` relies on a "growing" internal state.

Reply:
Can I see real life usage example?